### DISCUSSION TITLE
Adding a "Select all" button to tables

### DISCUSSION THREAD

**Discussion by samyxdev** [langfuse/langfuse#1394] [Ideas]
### Describe the feature or potential improvement

Adding a button to select more than the visible number of traces (eg. "selected 50 out of 1352829 traces, do you want to select all?")

**Comment by clemra** [comment]
Thank you for writing in. I've added this to our weekly. Will update you here if and when we decide to prioritize it.

**Reply by samyxdev** [reply]
Amazing, thanks !

**Reply by marcklingen** [reply]
We had a deeper look at this, the current implementation of deleting multiple rows does not scale well to e.g. 1352829 traces, meaning it is not just UI to add to make this happen.

This is a great idea, will get back to this especially as adding more than 50 items is super valuable for the upcoming evaluation feature.

**Reply by samyxdev** [reply]
@marcklingen Understood, looking forward to see that implemented.
